src/main/java/wolox/training/models/Users.java

```diff
@@ -61,6 +66,8 @@ public LocalDate getBirthdate() {
     }
 
     public void setBirthdate(LocalDate birthdate) {
+
+        Preconditions.checkNotNull(birthdate, "Please check the LocalDate birthdate supplied, its null!");
```

Comment:
A good validation would be to verify if birthdate is less than the current date.
😄